{{ cookiecutter.app_name }}/tests/test_app.py

```diff
@@ -0,0 +1,4 @@
+
+def test_first():
+    "An initial test for the app"
+    assert 1 + 1 == 2
```

Comment:
If we're supporting unittest as well, then there should be a template with an example test for that too.

Reply:
🤦 Obvious problem is obvious. Fixed.